index.js

```diff
@@ -124,7 +124,14 @@ function createMainWindow() {
 	remote.enable(win.webContents);
 	if (windowPosition) {
 		const { x, y } = windowPosition;
-		win.setPosition(x, y);
+		if(x + win.getSize()[0] < 0 || x - win.getSize()[0] > electron.screen.getDisplayNearestPoint({x, y}).bounds.width) {
```

Comment:
No need to do the same with y/height?

Reply:
Well usually the error occurs because x value is offscreen but just to make sure ill add it